strings.go

```diff
@@ -14,17 +14,31 @@ import (
 func obfuscateLiterals(files []*ast.File) []*ast.File {
 	pre := func(cursor *astutil.Cursor) bool {
 		decl, ok := cursor.Node().(*ast.GenDecl)
-		if !ok || decl.Tok != token.CONST {
+		if !ok {
```

Comment:
you should still check if this is CONST or VAR; you don't care about TYPE and IMPORT

Reply:
Right